src/components/ListedSpecies/BehaviorsSection.jsx

```diff
@@ -1,10 +1,12 @@
-import { FormControl, FormLabel, Grid, GridItem } from '@chakra-ui/react';
+import { FormControl, FormLabel, Grid, GridItem, Text } from '@chakra-ui/react';
+import { Select } from 'chakra-react-select';
 import { PropTypes } from 'prop-types';
 import React from 'react';
 import ReactHookFormSelect from '../../common/ReactHookFormSelect';
 import CollapsibleSection from '../CollapsibleSection/CollapsibleSection';
 
-const BehaviorsSection = ({ behaviorOptions, nestingOptions }) => {
+const BehaviorsSection = ({ behaviorOptions, nestingOptions, isTemplate }) => {
+  const { control } = useFormContext();
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'control' is assigned a value but never used.